tutorials/3d/introduction_to_3d.rst

```diff
@@ -44,7 +44,7 @@ little more difficult. The content needs to be created with special 3D tools
 exchange file format to be imported in Godot. This is required since 3D formats
 are not as standardized as images.
 
-Maually authored models (using 3D modeling software)
+Manually authored models (using 3D modeling software)
 ----------------------------------------------------
```

Comment:
```suggestion
-----------------------------------------------------
```
The number of hyphens needs to match the number of characters in the line above. Otherwise this looks great